README.md

```diff
@@ -74,6 +74,13 @@ Example user story: `As an admin I want to be able to create new users because I
 
 _These guidelines for creating user stories are an adaptation of [Simon's](https://github.com/simonlab) [**user-story-checklist**](https://github.com/SimonLab/user-story-checklist) repo. Check it out!_
 
+### Pull Requests
```

Comment:
Oooooooh, this whole section actually reminds me of: https://github.com/dwyl/style-guide#pull-requests

I'm not even sure which repo all of this should be in... @nelsonic ?